### PR TITLE
Updated PHPUnit version

### PR DESCRIPTION
**Pull request by DonCallisto**

### CODE REVIEW

File: tests/Badge/Service/TextNormalizerTest.php [hunk 1]
```diff
@@ -28,8 +28,10 @@ protected function setUp(): void
 
     /**
      * @dataProvider getBadNumberToConvert
+     *
+     * @phpstan-param int|string $input
      */
-    public function testNumberToTextConversion(string $input, string $output): void
+    public function testNumberToTextConversion($input, string $output): void
```
Comment (JellyBellyDev) [inline tests/Badge/Service/TextNormalizerTest.php:34]: in which use case do we receive a int?

**Comment by DonCallisto** [comment]
One thing I was forgetting: I've noticed an inconsistent way to create mocks in tests; sometimes is used built-in PHPUnit functionality, sometimes is used Prophecy. I think we should stick with one or the other. 
Indeed it's not a big issue, because no problem would arise, but I can see it like a "coding standard"; maybe could be worthy to "standardize" this as well. WDYT?

**Review by JellyBellyDev — CHANGES_REQUESTED** [review]
Good works!

File: tests/Badge/Service/TextNormalizerTest.php (hunk 1 above)
Comment (DonCallisto) [reply tests/Badge/Service/TextNormalizerTest.php:34]: https://github.com/PUGX/badge-poser/blob/d3f75e4f3a554708bb3515cb32a85ebb588d8939/tests/Badge/Service/TextNormalizerTest.php#L49

Don't know if this scenario is possible, but the goal of this PR was to restore codecoverage, not to "fix" unconsistent tests. If an `int` won't ever be possible in this situation, we should remove it from data provider (another issue/PR from my standpoint).

Comment (JellyBellyDev) [reply tests/Badge/Service/TextNormalizerTest.php:34]: I think that if the use case is present in data provider then that's right as you did! 👍

**Review by JellyBellyDev — APPROVED** [review]
LGTM

**Comment by JellyBellyDev** [comment]
> One thing I was forgetting: I've noticed an inconsistent way to create mocks in tests; sometimes we use built-in PHPUnit functionality, other times Prophecy. I think we should stick with one or the other.
> Indeed it's not a big issue, because no problem would arise, but I can see it like a "coding standard"; maybe could be worthy to "standardize" this as well. WDYT?

This is a classic problem in big open source project! An issue could be opened to promote the standardization request!

**Pull request merged**